plugins/janus_sip.c

```diff
@@ -3162,6 +3164,17 @@ static void *janus_sip_handler(void *data) {
 				to = session->account.identity;
 			const char *event_type = json_string_value(json_object_get(root, "event"));
 			const char *accept = json_string_value(json_object_get(root, "accept"));
+
+			/* TTL */
+			int ttl = subscribe_ttl;
+			json_t *sub_ttl = json_object_get(root, "subscribe_ttl");
```

Comment:
New parameters need to be added to the validation object, in this case `subscribe_parameters`.